app/gui/view/examples/interface/src/lib.rs

```diff
@@ -254,6 +254,13 @@ fn init(app: &Application) {
     graph_editor.set_node_profiling_status(node3_id, node3_status);
 
 
+    // === Execution Modes ===
+
+    graph_editor.set_execution_modes(vec!["development".to_string(), "production".to_string()]);
```

Comment:
These strings ar defiend in another file as well

Reply:
These are dummy strings anyway. The ones in the demo scene will only be used there, while the other ones will be removed during integration.